tests/frontend/helper.js

```diff
@@ -93,6 +93,20 @@ var helper = {};
   // This ensures that tests run regardless of this problem
   helper.retry = 0
 
+  helper.rootPath = '/p/';
```

Comment:
what if for some reason someone isn't using /p/ but instead has pads on root?

https://mypads.whatever.com/thisIsAPad for example...

Reply:
They use a proxy for this purpose and if they are configured correctly there is no problem.

Reply:
They will be configured as per the documentation: https://github.com/ether/etherpad-lite/wiki/How-to-put-Etherpad-Lite-behind-a-reverse-Proxy -- For example see https://github.com/ether/etherpad-lite/wiki/How-to-put-Etherpad-Lite-behind-a-reverse-Proxy#add-this-to-allow-nice-urls-such-as-httpsetherpadexampleorgpadname -- note that it includes support for not including a /p/ ...    :+1: